test/test_relational.py

```diff
@@ -54,8 +54,9 @@
 from lale.operators import make_pipeline_graph
 
 # Changing the current working directory to read the Go-Sales dataset for testing purposes
```

Comment:
This comment no longer applies (which is a good thing) so it should be removed as well

Reply:
Done!